src/components/MainComponent.js

```diff
@@ -17,6 +17,26 @@ import Debug from 'components/Debug'
 import { isChromeExtensionBackground } from 'services/notificationService'
 import Notifications from 'components/Extension/Notifications'
 
+try {
+  const Checklist = require('components/Checklist')
+}
+catch(error) {
+  if ( !error.toString().includes('Cannot find module') ) {
+    throw error
+  } else {
+
+  }
+}
+
+const isPluginDefined = () => {
+  if ( typeof Checklist !== undefined ) {
+    return false
```

Comment:
I think the conditional is backwards. Should return true if not undefined:
`const isPluginDefined = () => typeof Checklist !== undefined`